source/manual/deploying-terraform.html.md

```diff
@@ -4,76 +4,82 @@ title: Deploy AWS infrastructure with Terraform
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-08-04
+last_reviewed_on: 2019-11-08
 review_in: 3 months
 ---
 
-We use [Terraform](https://terraform.io) for configuring the GOV.UK
-infrastructure in AWS.
+We use [Terraform](https://terraform.io) for configuring GOV.UK infrastructure in AWS.
+This document describes:
 
-## 1. Check what you can deploy
+ * How to set up your machine to run Terraform.
+ * How to deploy infrastructure changes using Terraform.
+
+## One-time setup
```

Comment:
@issyl0 I think it's worth extracting this section to a separate page as it's not actually about deploying terraform. We can then add link to that page here with something like "if you haven't done this before, read the one-time setup doc page".

Reply:
It *is* though. It's about using our tooling to deploy Terraform, for which you need to do some setup.

Reply:
@sengi What do you think here?

Reply:
I think it's a fair comment but I can see arguments for/against either way. I think there's some merit in reducing the number of different documents that a new user has to navigate through in order to complete the setup process. On the other hand, I accept that keeping both the setup steps and the "doing the thing" steps in this document means that the document kinda has two purposes.

I think I have a mild preference for keeping it as-is, but I could be persuaded otherwise if you feel strongly about it.